docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,527 @@
+    SampleAppGenesisEtcdRaft:
+        <<: *ChannelDefaults
+        Orderer:
+            <<: *OrdererDefaults
+            OrdererType: etcdraft
+            Organizations:
+                - <<: *OrdererOrg
+                  Policies:
+                      <<: *OrdererOrgPolicies
+                      Admins:
+                          Type: Signature
+                          Rule: "OR('OrdererOrg.member')"
+        Application:
+            <<: *ApplicationDefaults
+            Organizations:
+                - <<: *SampleOrg
+                  Policies:
+                      <<: *SampleOrgPolicies
+                      Admins:
+                          Type: Signature
+                          Rule: "OR('SampleOrg.member')"
+
+```
+
+**Note:** This snippet assumes that you have also defined the `SampleOrg` in the `Organizations:` section of the configtx.yaml. Refer to the [sample config](https://github.com/hyperledger/fabric/blob/{BRANCH}/sampleconfig/configtx.yaml) for an example.
+
+You can create multiple channel profiles according to your privacy needs by including different combinations of peer and orderer organizations. In the next step, you specify which profile to use when you generate the channel genesis block.
+
+### Generate the genesis block
+
+After you have completed editing the `configtx.yaml`, you can start to create a new channel for the peer organizations.  Every channel configuration starts with a genesis block.  Because we previously set the environment variables for the `configtxgen` tool, you can run the following command to build the genesis block for `channel1` using the `SampleAppGenesisEtcdRaft` profile:
+
+```
+configtxgen -profile SampleAppGenesisEtcdRaft -outputBlock ./channel-artifacts/channel1.tx -channelID channel1
+```
+
+Where:
+- `-profile` -  Set this value to the name of the profile to use from `configtx.yaml`.
+- `-outputBlock` - Provide the location of where to store the generated configuration block file.
+- `-channelID` - Specify the name of the channel you want to create. Channel names must be all lower case, less than 250 characters long and match the regular expression ``[a-z][a-z0-9.-]*``. The command uses the `-profile` flag to reference the `SampleAppGenesisEtcdRaft:` profile from `configtx.yaml`.
+
+
+When the command successful, you will see logs of `configtxgen` loading the `configtx.yaml` file and printing a channel creation transaction:
+```
+2020-03-11 16:37:12.695 EDT [common.tools.configtxgen] main -> INFO 001 Loading configuration
+2020-03-11 16:37:12.738 EDT [common.tools.configtxgen.localconfig] Load -> INFO 002 Loaded configuration: /Usrs/fabric-samples/test-network/configtx/configtx.yaml
+2020-03-11 16:37:12.740 EDT [common.tools.configtxgen] doOutputChannelCreateTx -> INFO 003 Generating new channel configtx
+2020-03-11 16:37:12.789 EDT [common.tools.configtxgen] doOutputChannelCreateTx -> INFO 004 Writing new channel tx
+```
```

Comment:
This looks like it was generated by the -createChannelTx command and not -outputBlock. :)

Reply:
Changed it to:
INFO 001 Loading configuration
INFO 002 orderer type: etcdraft
INFO 003 Loaded configuration: /fabric/config/configtx.yaml
INFO 004 Generating genesis block
INFO 005 Writing genesis block